spec.html

```diff
@@ -48962,88 +48934,89 @@ <h1>agent-order</h1>
       </emu-note>
 
       <emu-note>
-        <p>~init~ events do not participate in synchronizes-with, and are instead constrained directly by happens-before.</p>
+        <p>In a candidate execution _execution_, ~init~ events do not participate in this Relation and are instead constrained directly by happens-before.</p>
       </emu-note>
 
       <emu-note>
-        <p>Not all ~seq-cst~ events related by reads-from are related by synchronizes-with. Only events that also have equal ranges are related by synchronizes-with.</p>
+        <p>In a candidate execution _execution_, not all ~seq-cst~ events related by reads-from are related by synchronizes-with. Only events that also have equal ranges are related by synchronizes-with.</p>
       </emu-note>
 
       <emu-note>
-        <p>For Shared Data Block events _R_ and _W_ such that _W_ synchronizes-with _R_, _R_ may reads-from other writes than _W_.</p>
+        <p>For Shared Data Block events _R_ and _W_ in a candidate execution _execution_ such that _W_ synchronizes-with _R_, _R_ may reads-from other writes than _W_.</p>
       </emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-happens-before" aoid="happens-before">
+    <emu-clause id="sec-happens-before">
       <h1>happens-before</h1>
-      <p>For a candidate execution _execution_, _execution_.[[HappensBefore]] is the least Relation on events that satisfies the following.</p>
+      <p>For a candidate execution _execution_, its <dfn>happens-before</dfn> Relation is the least Relation on events that satisfies the following.</p>
 
       <ul>
-        <li>For each pair (_E_, _D_) in _execution_.[[AgentOrder]], _execution_.[[HappensBefore]] contains (_E_, _D_).</li>
-        <li>For each pair (_E_, _D_) in _execution_.[[SynchronizesWith]], _execution_.[[HappensBefore]] contains (_E_, _D_).</li>
-        <li>For each pair (_E_, _D_) in SharedDataBlockEventSet(_execution_), _execution_.[[HappensBefore]] contains (_E_, _D_) if _E_.[[Order]] is ~init~ and _E_ and _D_ have overlapping ranges.</li>
-        <li>For each pair (_E_, _D_) in EventSet(_execution_), _execution_.[[HappensBefore]] contains (_E_, _D_) if there is an event _F_ such that the pairs (_E_, _F_) and (_F_, _D_) are in _execution_.[[HappensBefore]].</li>
+        <li>For each pair (_E_, _D_) in is-agent-order-before in _execution_, _E_ happens-before _D_ in _execution_.</li>
+        <li>For each pair (_E_, _D_) in synchronizes-with in _execution_, _E_ happens-before _D_ in _execution_.</li>
+        <li>For each pair (_E_, _D_) in SharedDataBlockEventSet(_execution_), _E_ happens-before _D_ in _execution_ if _E_.[[Order]] is ~init~ and _E_ and _D_ have overlapping ranges.</li>
+        <li>For each pair (_E_, _D_) in EventSet(_execution_), _E_ happens-before _D_ in _execution_ if there is an event _F_ such that _E_ happens-before _F_ and _F_ happens-before _D_ in _execution_.</li>
```

Comment:
Again, like above, I find this would read much better:

```suggestion
        <li>
          For each event _E_ and _D_, _E_ happens-before _D_ in _execution_ if any of the following hold:
          <ul>
            <li>_E_ is-agent-order-before _D_ in _execution_</li>
            <li>_E_ synchronizes-with _D_ in _execution_</li>
            <li>SharedDataBlockEventSet(_execution_) contains (_E_, _D_), _E_.[[Order]] is ~init~, and _E_ and _D_ have overlapping ranges.</li>
            <li>EventSet(_execution_) contains (_E_, _D_) and for some event _F_, _E_ happens-before _F_ in _execution_ and _F_ happens-before _D_ in _execution_.</li>
          </ul>
        </li>
```